pkg/batchrunner/batch_action_runner.go

```diff
@@ -14,28 +14,60 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package batchrunner
 
 import (
 	"fmt"
 
 	"github.com/golang/glog"
-	batch "k8s.io/api/batch/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-
 	alertmanager "github.com/openshift/autoheal/pkg/alertmanager"
 	"github.com/openshift/autoheal/pkg/apis/autoheal"
+	batch "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/kubernetes"
 )
 
-func (h *Healer) runBatchJob(rule *autoheal.HealingRule, job *batch.Job, alert *alertmanager.Alert) error {
+type BatchActionRunnerBuilder struct {
+	k8sClient kubernetes.Interface
+	stopCh    <-chan struct{}
+}
+
+type BatchActionRunner struct {
+	k8sClient kubernetes.Interface
+}
+
+func NewBatchActionRunnerBuilder() *BatchActionRunnerBuilder {
+	return new(BatchActionRunnerBuilder)
+}
+
+func (b *BatchActionRunnerBuilder) KuberenetesClient(k8sClient kubernetes.Interface) *BatchActionRunnerBuilder {
+	b.k8sClient = k8sClient
+	return b
+}
+
+func (b *BatchActionRunnerBuilder) StopCh(stopCh <-chan struct{}) *BatchActionRunnerBuilder {
+	b.stopCh = stopCh
+	return b
+}
+
+func (b *BatchActionRunnerBuilder) Build() *BatchActionRunner {
```

Comment:
The build method should return the built object an and an error. We may not need the error today, but it will most likely needed in the future. Please add it.